frontend/src/components/ArtistHeader.js

```diff
@@ -6,8 +6,8 @@ export default function ArtistHeader({artist}) {
     return (
         <Link to={`/artist/${artist.artistName}`} style={{textDecoration: 'none'}}>
             <Header>
-                <img src={artist.artistInfo.artistImageUrl} alt={'Shows the artist'}/>
-                <h3>{artist.artistName}</h3>
+                <img src={artist.artistInfo.artistImageUrl} alt={'not available'}/>
```

Comment:
`alt` attribute should contain a string that describes what the image shows, if the image is only there for aesthetic reasons, set `alt=""` explicitley.

Reply:
ok